### DISCUSSION TITLE
.md to .html conversion does not seem to work properly

### DISCUSSION THREAD

**Discussion by flemmingss** [community/community#22095] [Other Feature Feedback, Questions, & Ideas]
<p>Ok, this is my github md file in repo:</p>
<p><a href="https://github.com/flemmingss/flemmingss.github.io/blob/master/tutorials/deluge-on-ubuntu-server.md" rel="nofollow noopener">https://github.com/flemmingss/flemmingss.github.io/blob/master/tutorials/deluge-on-ubuntu-server.md</a></p>
<p>when displayed in github it all works.</p>
<p>But when i show it in “pages” some of the formating are not displayed.</p>
<p><a href="https://flemmingss.github.io/tutorials/deluge-on-ubuntu-server.html" rel="nofollow noopener">https://flemmingss.github.io/tutorials/deluge-on-ubuntu-server.html</a></p>
<p>As you se, instad of converting to html it display the markup taggs like normal content.</p>
<p>How can i fix this? I want it to show correct both in the repositorie and page.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I’ve submitted <a href="https://github.com/flemmingss/flemmingss.github.io/pull/1" rel="nofollow noopener">a Pull Request</a> with some changes I recommend.</p>
<p>The reason for the differences in rendering is because <a href="http://github.com" rel="nofollow noopener">github.com</a> uses <a href="https://commonmark.org" rel="nofollow noopener">CommonMark</a> for its Markdown rendering whereas GitHub Pages uses Jekyll and <a href="https://kramdown.gettalong.org/" rel="nofollow noopener">Kramdown</a> for its Markdown rendering. Because the two components use different rendering engines, there are differences in interpretation of Markdown documents. However, incorporating the changes I suggested in the PR above should take care of those problems for you.</p>
<p>I hope that helps!</p>

**Comment by lee-dohm** [comment]
<p>A post was split to a new topic: <a href="https://github.community/t/rendering-of-a-specific-page-doesnt-seem-to-work/123704">Rendering of a specific page doesn’t seem to work?</a></p>
